package.json

```diff
@@ -5,7 +5,7 @@
   "private": true,
   "engines": {
     "npm": ">=8",
-    "node": ">=16.*"
+    "node": ">=18.*"
```

Comment:
Should we match the template project style for these (I know they mean the same)

"engines": {
    "node": "^18",
    "npm": "^8"
  }